src/js/components/useToast.test.ts

```diff
@@ -37,92 +37,94 @@ describe('useToast', () => {
 
     it('should display the HTML markup message when used', () => {
       const toast = useToast(Toastify.TestMarkupMessage);
-      toast.instance.show();
-
-      const toastMessage = toast.content.innerHTML;
+      toast.show();
+      const toastMessage = toast.content();
       const expectedToastMessage = Toastify.TestMarkupMessage;
-      toast.element.remove();
+      toast.remove();
 
       expect(toastMessage === expectedToastMessage).toBeTruthy();
     });
 
     it('should style the toast as same as the type when is set', () => {
       const toast = useToast('', Toastify.TestTypeOption);
-      toast.instance.show();
-
-      const toastClassList = toast.element.classList.toString();
+      toast.show();
+      const toastElement = toast.element();
```

Comment:
```suggestion
      const toastElement = toast.element();
```

Why would we need `toast.element()` while you can get the toast element using destructuring when you initialize the toast like

```
const {element} = useToast(...)
```